x-pack/plugins/observability/README.md

```diff
@@ -4,33 +4,14 @@ This plugin provides shared components and services for use across observability
 
 ## Rules, Alerts, and Cases
 
-The Observability plugin contains experimental support for improved alerting and
-case management.
-
-If you have:
-
-```yaml
-xpack.observability.unsafe.cases.enabled: true
-```
-
-In your Kibana configuration, the Cases page will be available.
-
-If you have:
-
-```yaml
-xpack.observability.unsafe.alertingExperience.enabled: true
-```
-
-In your Kibana configuration, the Alerts page will be available.
-
-This will only enable the UI for this page when. In order to have alert data indexed
+In order to have alert data indexed
```

Comment:
@mgiota, the config key `xpack.observability.unsafe.rules.enabled` wasn't mentioned in the O11y readme?

Reply:
@fkanout Yep this flag wasn't mentioned in the o11y readme, because I introduced it mainly for internal purposes. Meaning that while new rules page was in development, I wanted to have it disabled by default, so that it didn't look broken for other people, but still wanted to have the code available on main, in case more people needed to work in parallel on the new page. Hope that makes sense!